src/components/buttons/base-button-component/base-button-component.ts

```diff
@@ -90,4 +91,9 @@ export class BaseButtonComponent {
   public getElement(): HTMLElement {
     return this.element;
   }
+
+  public _destroy() {
```

Comment:
rename to `destroy()`.

Let's keep this consistent, eventually all html components will extend the base html one.